### PR TITLE
fix: set default barrier interval to 1s

### PR DESCRIPTION
**Pull request by fuyufjh**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

Set the default barrier interval to 1s, which is used by our cloud service.

Meanwhile, keep the testing config as 250ms.

The config in `risingwave.toml` was not changed, so I suppose our e2e test will also be run in 250ms interval. 

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

It seems that we didn't mention anything about checkpointing in docs...

## Refer to a related PR or issue link (optional)

None

### CODE REVIEW

**Comment by BugenZhao** [comment]
> The config in `risedev.yml` was not changed

Do you mean `risingwave.toml`? 👀

**Comment by fuyufjh** [comment]
> > The config in `risedev.yml` was not changed
> 
> Do you mean `risingwave.toml`? 👀

Yes 🤣

**Comment by BugenZhao** [comment]
It looks like the scaling simulation test requires a more frequent barrier interval, while it doesn't follow the config in `risingwave.toml`... 🤔 cc @wangrunji0408

**Comment by wangrunji0408** [comment]
> It looks like the scaling simulation test requires a more frequent barrier interval, while it doesn't follow the config in `risingwave.toml`... 🤔 cc @wangrunji0408

Yes, it doesn't use the config from `risingwave.toml` because it has a larger parallelism setting which makes the simulation very slow. 😰

**Comment by fuyufjh** [comment]
> > It looks like the scaling simulation test requires a more frequent barrier interval, while it doesn't follow the config in `risingwave.toml`... 🤔 cc @wangrunji0408
> 
> Yes, it doesn't use the config from `risingwave.toml` because it has a larger parallelism setting which makes the simulation very slow. 😰

Is it possible to inject the config via file in simulation test?

https://github.com/risingwavelabs/risingwave/blob/baddd63f275b10c6816e66f96ccf24c52b1b61d5/src/tests/simulation_scale/src/cluster.rs#L80-L86

**Comment by BugenZhao** [comment]
The recovery test only uses 3m54s under this configuration. 🤯 cc @yezizp2012

**Review by hzxa21 — APPROVED** [review]

**Comment by yezizp2012** [comment]
> The recovery test only uses 3m54s under this configuration. 🤯 cc @yezizp2012

I guess by changing barrier interval from 250ms to 1s, barrier-related simulations will be 3/4 less. Does it make sense ? @wangrunji0408 🤔

**Comment by wangrunji0408** [comment]
> > The recovery test only uses 3m54s under this configuration. 🤯 cc @yezizp2012
> 
> I guess by changing barrier interval from 250ms to 1s, barrier-related simulations will be 3/4 less. Does it make sense ? @wangrunji0408 🤔

Make sense!

File: src/common/src/config.rs [hunk 1]
```diff
@@ -365,7 +365,7 @@ mod default {
     }
 
     pub fn barrier_interval_ms() -> u32 {
-        250
+        1000
```
Comment (BugenZhao) [inline src/common/src/config.rs:368]: Should we note somewhere (in `toml`?) that the default value without configuration file here differs from the default value in `risingwave.toml`? 🥵

**Review by BugenZhao — APPROVED** [review]
LGTM. I've added a separate configuration file for the simulation scaling test and hope it works.

**Comment by yezizp2012** [comment]
> > > The recovery test only uses 3m54s under this configuration. 🤯 cc @yezizp2012
> > 
> > 
> > I guess by changing barrier interval from 250ms to 1s, barrier-related simulations will be 3/4 less. Does it make sense ? @wangrunji0408 🤔
> 
> Make sense! ~In fact all simulations may benefit.~ 🥵

The `end-to-end-test-deterministic-simulation` also reduced from 9min+ to 7min+. 🥵

**Review by yezizp2012 — APPROVED** [review]

**Pull request merged**